results.py

```diff
@@ -193,14 +193,19 @@
     "sentence-t5-large",
     "sentence-t5-xl",
     "sentence-t5-xxl",
-    "sentence-transformers__LaBSE",
-    "sentence-transformers__all-MiniLM-L12-v2",
```

Comment:
I see this model carried over -- did we just have all the other ones already? Like all-MiniLM-L6-v2?

Reply:
Did we add these back in -why was this resolved?

Reply:
These models was already there

Reply:
Right, but now we use the outdated version instead of the latest version of the results (which should be the same). I also believe that these new folders contain more results. It is not a big issue for now as we have plans to update the leaderboard, but it is worth noting the issues.